README.md

```diff
@@ -1,2 +1,19 @@
-# HW01
-Edit the readme
+# Hello Class!
+
+My name is Daniel and I'm from Poland. I arrived to the US about three years ago for an internship at UChicago. Afterwards I decided to stay in this beautiful country for a bit longer. Now I'm happy to be starting a PhD studies on the BMB program here!
+
+I was born in Poland in a city of Bielsko-Biala, located in southern Poland - medium-sized (for polish standards) and surrounded by old, green mountains, that greatly add to its charm.
```

Comment:
This is awesome! I'm Polish too - my mom's family lives in Krosno, and my dad's lives in Knurów.

Reply:
That's really cool, so you have roots in southern Poland! And there are some great places to visit near Krosno. Thanks for the review and hoping to meet you in person soon!